src/tinysplinecxx.h

```diff
@@ -4,6 +4,14 @@
 #include <vector>
 #include <string>
 
+#ifdef TINYSPLINECXX_STATIC
```

Comment:
__declspec(dllexport) is not required here?

Reply:
No, this allows using tinyspline inside a shared library without exporting the symbols.

Reply:
I see.